src/Jackett.Common/Indexers/PolishTracker.cs

```diff
@@ -152,17 +155,23 @@ protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuer
                             poster = new Uri(CdnUrl + "images/torrents/poster/ste/l/" + torrent["steam_id"] + ".jpg");
                     }
 
+                    var title = torrent.name.ToString();
+
                     var descriptions = new List<string>();
                     var language = (string)torrent.language;
                     if (!string.IsNullOrEmpty(language))
                         descriptions.Add("Language: " + language);
                     else if ((bool?)torrent.polish == true)
                         descriptions.Add("Language: pl");
+
+                    if (language == "pl" && (((BoolItem) configData.GetDynamic("LanguageTitle")).Value))
```

Comment:
In the previous line I see `torrent.polish == true`
Could be better to check that instead language or it's the same?

Reply:
Yes, i thinks this same, but i check and all torents has ```
language == "pl"
``` but havent ```torrent.polish == true```